dm/pkg/checker/utils.go

```diff
@@ -135,7 +136,12 @@ func markCheckError(result *Result, err error) {
 		if result.State != StateFailure {
 			result.State = state
 		}
-		result.Errors = append(result.Errors, &Error{Severity: state, ShortErr: err.Error()})
+		if err2, ok := err.(*terror.Error); ok {
+			result.Errors = append(result.Errors, &Error{Severity: state, ShortErr: err2.ErrorWithoutWorkaround()})
+			result.Instruction = err2.Workaround()
```

Comment:
How does it look like?

Reply:
![image](https://user-images.githubusercontent.com/3312245/217701761-f60792db-c570-4c4a-bcb8-a145324c228b.png)

Reply:
Can we trim the `[....],` from the `short_error`? Looks confusing.

Reply:
https://github.com/pingcap/tiflow/pull/8201/commits/4f082b8641c325ddc4c7f15ffa117f0fab423e57

![image](https://user-images.githubusercontent.com/3312245/217708398-08c8b1ca-bc53-443b-99a6-0362f5632b03.png)

Reply:
The `Errors` may contain multiple errors, so instruction should also be prepraed for multiple lines?

Reply:
precheck item returns on first error meet. if error passed into it is a multi-err instance(tables check), use the `else` branch.